lessons/1_setup/index.html

```diff
@@ -1,154 +1,212 @@
+              <b>In-person lesson:</b>
+              <ul>
+                <li class="fragment"><b>Where?</b>SIL</li>
+                <li class="fragment">
+                  <b>When?</b>19:00 - 21:00 Friday every two week
+                </li>
+              </ul>
+            </li>
+          </ul>
+        </section>
+        <section>
+          <h3>What is expected from you?</h3>
+          <ul>
+            <li class="fragment">Commitment</li>
+            <li class="fragment">Desire to learn</li>
+            <li class="fragment">Patience</li>
+            <li class="fragment">Code-code-code</li>
+          </ul>
+        </section>
+        <section>
+          <h3>What you can expect from us?</h3>
+          <ul>
+            <li class="fragment">
+              Help inside and outside of class hours, via Slack chat or calls
+            </li>
+            <li class="fragment">Project inspiration and suggestions</li>
+          </ul>
+        </section>
+        <section>
+          <h3>The Final Project</h3>
+          <ul>
+            <li class="fragment">Each of you has to finish a small project</li>
+            <li class="fragment">
+              The projects will be presented on demo day to all from ReDI Munich
+              and guests
+            </li>
+            <li class="fragment">
+              You have lots of freedom how to design and build it
+            </li>
+            <li class="fragment">
+              Start with small simple version and then add more features
+            </li>
+            <li class="fragment">We will help you every step of the way!</li>
+          </ul>
+        </section>
+        <section>
+          <h3>What could be a Final Project</h3>
+          <ul>
+            <li class="fragment">Weather check app</li>
+            <li class="fragment">Stock price check page</li>
+            <li class="fragment">Movie app</li>
+            <li class="fragment">Todo app</li>
+            <li class="fragment">Twitter clone</li>
+            <li class="fragment">Snake game</li>
+            <li class="fragment">... bring your own idea!</li>
+          </ul>
+        </section>
+        <section>
+          <h3>Laptop checkup</h3>
+          <ul class="fragment">
+            <li>
+              Firefox:
+              <a href="https://www.mozilla.org/firefox/new/"
+                >https://www.mozilla.org/firefox/new/</a
+              >
+            </li>
+            <li>
+              VSCode*:
+              <a href="https://code.visualstudio.com/download"
+                >https://code.visualstudio.com/download</a
+              ><small>* ask us about the recommended plugins</small>
+            </li>
+            <li>
+              Github: <a href="https://github.com/">https://github.com/</a>
+            </li>
+            <li>
+              freeCodeCamp:
+              <a href="https://www.freecodecamp.org/"
+                >https://www.freecodecamp.org/</a
```

Comment:
Just curious: is it Prettier who format it's like that?)

Reply:
haha yeah. do you use a diff one?

Reply:
this is why I'm not using it, because after that the code looks, let's say, strange :)

Reply:
haha I understand, nevertheless: https://prettier.io/docs/en/why-prettier.html 

when it comes to styling I'm more of an adopter than a trendsetter :)

Reply:
Maybe I should give it another try ;)